NewDawn/NewDawn/ViewControllers/RegistrationViewControllers/Profile_DraftFinal.swift

```diff
@@ -68,6 +69,7 @@ class Profile_DraftFinal: UIViewController {
             }
             DispatchQueue.main.async {
                 self.performSegue(withIdentifier: "after_register", sender: self)
+                Mixpanel.mainInstance().track(event: "Registration Start")
```

Comment:
Should this be Registration Completed?

Reply:
Their names need to be the same. I just changed both to "registration duration".